functionary/core/models/team.py

```diff
@@ -24,4 +24,4 @@ def __str__(self):
     def post_create(self):
         """Post create hooks"""
         Environment = apps.get_model("core", "Environment")
-        Environment.create_default(self)
+        Environment.create_first_env(self)
```

Comment:
The original reason for the `create_default` method was that I didn't want the "Team" model to have to know the implementation details of declaring an Environment as the "default".  Not that those details were especially complex.

Given that there's nothing special about this initial environment that gets created now, I think it's okay to just create it here and delete get rid of the whole `create_default` / `create_first_env` method.  So this line could just become:

```suggestion
        Environment.objects.create(name="default", team=self)
```